jupyter/d2l-java/chapter_convolutional-neural-networks/lenet.ipynb

```diff
@@ -246,19 +228,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 18,
    "metadata": {},
    "outputs": [],
    "source": [
     "NDArray X = manager.randomUniform(0f, 1.0f, new Shape(1, 1, 28, 28));\n",
     "block.initialize(manager, DataType.FLOAT32, X.getShape());\n",
```

Comment:
You don't need this or the `block.setInitializer` because you initialize down in the trainer. Also, Xavier (or a more recent variant of it) is the default initializer in the TrainingConfig so you don't need to explicitly set it

Reply:
So the block gives me an error in the next code block (where I'm printing the output shapes) if I don't initialize it.